Content.Shared/Construction/Conditions/NoUnstackableInTile.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Content.Shared.Maps;
+using Content.Shared.Tag;
+using JetBrains.Annotations;
+using Robust.Shared.Map;
+
+namespace Content.Shared.Construction.Conditions
+{
+    [UsedImplicitly]
+    [DataDefinition]
+    public sealed class NoUnstackableInTile : IConstructionCondition
+    {
+        public bool Condition(EntityUid user, EntityCoordinates location, Direction direction)
+        {
+            var tagSystem = IoCManager.Resolve<IEntitySystemManager>().GetEntitySystem<TagSystem>();
+
+            foreach (var entity in location.GetEntitiesInTile(LookupFlags.Approximate | LookupFlags.Static | LookupFlags.Sundries))
+            {
+                if (tagSystem.HasTag(entity, "Unstackable"))
```

Comment:
Same here on the constant string.

Reply:
Same response as above. A constant just makes this code harder to understand AND conflicts with the rest of the codebase.